pkg/kubelet/kubelet_pods.go

```diff
@@ -1962,7 +1962,7 @@ func (kl *Kubelet) cleanupOrphanedPodCgroups(pcm cm.PodContainerManager, cgroupP
 		// parent croup.  If the volumes still exist, reduce the cpu shares for any
 		// process in the cgroup to the minimum value while we wait.  if the kubelet
 		// is configured to keep terminated volumes, we will delete the cgroup and not block.
-		if podVolumesExist := kl.podVolumesExist(uid); podVolumesExist && !kl.keepTerminatedPodVolumes {
+		if podVolumesExist := kl.podMountedVolumesExistInCacheOrDisk(uid); podVolumesExist && !kl.keepTerminatedPodVolumes {
```

Comment:
actually here cleanupOrphanedPodCgroups,I think it should be similar to cleanupOrphanedPod, so using podVolumePathsExistInCacheOrDisk. ?

Reply:
I changed this one to `kl. kl.podVolumePathsExistInCacheOrDisk`.

Right now no usage remain for the function `kl.podVolumesExist` or with the new name `kl.podMountedVolumesExistInCacheOrDisk`. Should I delete this function and all related test codes?